### ISSUE TITLE
Python 2 is close to deprecation

### ISSUE THREAD

**Issue by agarret7** [StanfordVL/NTP-vat-release#1]
Python 2 is close (1 year) to being deprecated. It took me maybe 20 minutes to update the code base to be compatible with Python 3, and it doesn't seem that there is any real necessity for using Python 2.

**Comment by danfeiX** [comment]
I agree. It was mostly because we had to deploy it to a real robot and ROS doesn't support Python3. I will update the code to py3 in a few days.
